pkg/monitor/cluster/nodeconditions.go

```diff
@@ -33,18 +32,20 @@ func (mon *Monitor) emitNodeConditions(ctx context.Context) error {
 			}
 
 			mon.emitGauge("node.conditions", 1, map[string]string{
-				"name":   n.Name,
-				"status": string(c.Status),
-				"type":   string(c.Type),
+				"name":           n.Name,
+				"kubeletVersion": n.Status.NodeInfo.KubeletVersion,
```

Comment:
this will not be emitted if the node conditions match nodeConditionsExpected.  Is that intentional?

Reply:
hmmm. I dont think so. but I think it would be enough this to be daily metric to avoid more data.

Reply:
TODO: Drop status checking from nodes